src/main/java/appeng/client/render/items/ToolWirelessTerminalRender.java

```diff
@@ -0,0 +1,126 @@
+package appeng.client.render.items;
+
+import appeng.api.util.AEColor;
+import appeng.client.texture.ExtraItemTextures;
+import appeng.items.tools.powered.ToolWirelessTerminal;
+import net.minecraft.client.renderer.ItemRenderer;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraftforge.client.IItemRenderer;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * Created by Tom on 9/30/2015.
+ */
+public class ToolWirelessTerminalRender implements IItemRenderer {
+
+    @Override
+    public boolean handleRenderType( ItemStack item, ItemRenderType type ) {
+        return true;
+    }
+
+    @Override
+    public boolean shouldUseRenderHelper( ItemRenderType type, ItemStack item, ItemRendererHelper helper ) {
+        return false;
+    }
+
+    @Override
+    public void renderItem( ItemRenderType type, ItemStack item, Object... data ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)